spec/acceptance/bootstrap_rpm_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper_acceptance'
 
-describe 'bootstrap_rpm' do
+describe 'bootstrap_rpm', :order => :defined do
```

Comment:
Should this remove all RPMs in `/var/www/html/pub/` before we start so you can run the test multiple times? If you now run rspec multiple times then it'll keep increasing the number.